test/e2e/run_test.go

```diff
@@ -1047,4 +1047,25 @@ USER mail`
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(strings.Contains(session.OutputToString(), groupName)).To(BeTrue())
 	})
+
+	It("podman run --tz", func() {
+		session := podmanTest.Podman([]string{"run", "--tz", "foo", "--rm", ALPINE, "date"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Not(Equal(0)))
+
+		session = podmanTest.Podman([]string{"run", "--tz", "Pacific/Honolulu", "--rm", ALPINE, "date", "+'%H %Z'"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		Expect(session.OutputToString()).To(ContainSubstring("HST"))
+
+		session = podmanTest.Podman([]string{"run", "--tz", "local", "--rm", ALPINE, "date", "+'%H %Z'"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		t := time.Now()
+		z, _ := t.Zone()
+		h := strconv.Itoa(t.Hour())
+		Expect(session.OutputToString()).To(ContainSubstring(z))
+		Expect(session.OutputToString()).To(ContainSubstring(h))
```

Comment:
Could you add a test with container.conf set to something and make sure it works.

Reply:
@rhatdan Do you know which test files has examples of a test with container.conf? I'm trying to find examples